File: Curso 1/Programación/Ejercicios/Ejercicios Java/coleccion4/arrays2/Ejercicio1.java
package coleccion4.arrays2;

/*
 * Título: Ejercicio 1
 * Algoritmo: Guardar los tamaños de las cadenas de un array
 * Fecha: 15.11.2022
 * Autor: Hugo Pelayo
 */

import java.util.Arrays;

public class Ejercicio1 {
    public static void main(String[] args) {
        String[] cadenas = new String[]{ "coche", "casa", "avión", "tren" };
        int[] tamanios = new int[cadenas.length];
        int indice = 0;

        for (String str : cadenas) {
            tamanios[indice++] = str.length();
        }

        System.out.println("Tamaños de las cadenas: " + Arrays.toString(tamanios));
    }
}

File: Curso 1/Programación/Ejercicios/Ejercicios Python/Ejercicios Python #7/Ejercicio1.py
# Ejercicio 1
# Algoritmo: Duplicar valores con lambda
# Fecha: 27.12.2022
# Autor: Hugo Pelayo

mi_lista = [-3, 4, 5, 1, 9]

nueva_lista = list(map(lambda numero: numero * 2, mi_lista))
indice = 0

for elemento in nueva_lista:
    print(f'Elemento en indice [{indice}]: {elemento}')
    indice += 1
